### PR TITLE
Devex 864: continue splitting Case unit tests into separate files

### PR DESCRIPTION
**Pull request by rachaelparris**

### CODE REVIEW

**Review by kkoskelin — APPROVED** [review]

File: shared/src/business/entities/cases/Case.unupdateAutomaticBlocked.test.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+const {
+  applicationContext,
+} = require('../../test/createTestApplicationContext');
+const { Case } = require('./Case');
+const { MOCK_CASE_WITHOUT_PENDING } = require('../../../test/mockCase');
+
+describe('unupdateAutomaticBlocked', () => {
```
Comment (rachelschneiderman) [inline shared/src/business/entities/cases/Case.unupdateAutomaticBlocked.test.js]: Nit: spelling, same with the filename 😄

File: shared/src/business/entities/cases/Case.updateAutomaticBlocked.test.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
+const {
+  applicationContext,
+} = require('../../test/createTestApplicationContext');
+const { AUTOMATIC_BLOCKED_REASONS } = require('../EntityConstants');
+const { Case } = require('./Case');
+const { MOCK_CASE } = require('../../../test/mockCase');
+
+describe('updateAutomaticBlocked', () => {
```
Comment (rachelschneiderman) [inline shared/src/business/entities/cases/Case.updateAutomaticBlocked.test.js:11]: Wait, is this the same as the previous test file?

Comment (rachaelparris) [reply shared/src/business/entities/cases/Case.updateAutomaticBlocked.test.js:11]: Ah, I'll combine them. Looks like they were split into separate describes for some reason. Good catch!

**Pull request merged**